### DISCUSSION TITLE
CHARGE flagged incorrectly 2.6.0 on Mac

### DISCUSSION THREAD

**Discussion by fatrabbit-la** [artisan-roaster-scope/artisan#906] [General]
Is anyone else having this issue? I've been using Artisan for several years and it has always flagged the CHARGE perfectly. Until this version. Now it consistently flags it a few seconds late and several degrees lower (as the cooler beans affect the BT probe).

I brought this up several months ago and Marko said nothing about Artisan changed that would affect this but something has definitely changed. If not Artisan, I'm wondering what it could possibly be because my setup has remained consistent. New computer, maybe? Updated OS?

I know I can easily CTRL-click on the graph to set the flag where it should be but as mentioned, I've never had to do this before and now I'm doing it for every roast. This often occurs with the DROP as well, but not as often.

I LOVE Artisan, but this is the one thing that's driving me bonkers.

**Comment by poundy** [comment]
just to confirm, you're USING or NOT USING the Autocharge feature? 

I am NOT using Autocharge. My previous roaster had high signal noise, and my BBP on my newer (less noisy, more efficient) setup means I am more often than not descending to charge, so it has never made sense for me to try putting it back on.

I'm not familiar with the code, but I expect Auto charge would use a specific rate change as it's trigger. If your probes have built up gunk and are responding slower, or your sample rate has changed at all, I can understand how that might lead to a delayed detection.

**Comment by fatrabbit-la** [comment]
Yes, I am using Autocharge and my signal is pretty clean. There is a very distinct rise in temperature based on my process for charging, followed by a distinct drop in temperature with an obvious peak showing the point at which I open the hopper (I lower the gas x amount and then raise it to my charge temp).

This probably wouldn't 'bug' me so much had I not been using previous versions of Artisan, literally for years, without this issue.

As far as I know, my sampling rate has always been the same at 3 seconds, Always On, though it's certainly possible with the new release that settings may have been changed to different defaults. But if so, I can't pinpoint what that would be as I haven't adjusted my settings through several updates. It's always just worked.
